src/proper.erl

```diff
@@ -556,29 +561,30 @@
 		  | {'to_file',io:device()}.
 
 -type user_opts() :: [user_opt()] | user_opt().
--record(opts, {output_fun       = fun io:format/2 :: output_fun(),
```

Comment:
Please avoid doing unnecessary (and excessive) cosmetic changes.
Moreover, in the code we are trying to stick to 80-column lines.